### DISCUSSION TITLE
Remote WebDriver

### DISCUSSION THREAD

**Discussion by Frohlix** [lewoudar/scalpel#14] [Ideas]
Hi :)

I'm working on a crawling project which will have to use Selenium for JavaScript reasons, but in the form of a standalone docker container like [standalone-firefox](https://hub.docker.com/r/selenium/standalone-firefox). If I'm reading [selenium.py](https://github.com/lewoudar/scalpel/blob/master/scalpel/core/selenium.py) correctly, scalpel does not support [Remote WebDrivers](https://www.selenium.dev/documentation/webdriver/remote_webdriver/), but it could work very similar to a local WebDriver. Do you think this might be a feature in the future?

**Comment by lewoudar** [comment]
Hi Frohlix,
thank you for your interest in this project. Yeah, it seems like a good idea to allow support for remote webdriver. I don't have time now to work on it, but maybe later in the year :)
